features/keychain/api/index.js

```diff
@@ -16,6 +16,7 @@ const createKeychainApi = ({ keychain }) => {
       },
       secp256k1: {
         signBuffer: keychain.secp256k1.signBuffer,
+        signSchnorr: keychain.secp256k1.signSchnorr,
```

Comment:
This should have a type added to index.d.ts as well.

Reply:
https://github.com/ExodusMovement/exodus-oss/pull/176

Reply:
Sorry, forgot to mention I added in #174.